include/libcyphal/presentation/presentation.hpp

```diff
@@ -74,7 +136,17 @@ class Presentation final : private detail::IPresentationDelegate
         return Publisher<Message>{publisher_impl};
     }
 
-    /// @brief Makes a message subscriber.
+    /// @brief Makes a typed publisher bound to its standard (aka fixed) subject id.
+    ///
+    /// @tparam Message The message type generated by DSDL tool. The type expected to have a fixed port ID.
+    ///
+    template <typename Message, typename Result = Expected<Publisher<Message>, MakeFailure>>
+    auto makePublisher() -> std::enable_if_t<detail::IsFixedPortIdMessageTrait<Message>::value, Result>
```

Comment:
API of `makePublisher` is also extended with automatic deduction of stuff if `Message` is "Nunavut" like.